CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * When emitting workflow outputs to the Cromwell log only the first 1000 characters per output will be printed
 * Added support for conditional (`if`) statements.
+* Globs for Shared File System (SFS) backends, such as local or SGE, now use bash globbing instead of Java globbing, consistent with the JES backend.
```

Comment:
`Consistent with "The WDL Spec"`. Also we should check that the WDL spec is consistent with "`The WDL Spec`"